### ISSUE TITLE
Error when trying to search

### ISSUE THREAD

**Issue by Shoulxe** [ruochenjia/whitespider-unblocker#2]
recently i went into the site to search something on my school laptop and i got "TypeError: Failed to fetch" instead of what i was searching for

if you know anything about this do you think its possible to fix

**Comment by Shoulxe** [comment]
tried again a day later and still getting an error

**Comment by deadpool098778** [comment]
Yeah man same here, whenever i try to search some it doesn't work and it doesn't work with my other friends either just tryna play cookie clicker xD

**Comment by Shoulxe** [comment]
sorry havent been on my laptop all weekend ill check out the console

**Comment by Shoulxe** [comment]
- oops

**Comment by Shoulxe** [comment]
ill take a look at the console i havent been on during the weekend

**Comment by Shoulxe** [comment]
ill have to check it out when i get home as my school laptop is so locked down i cant even access the inspect menu
